README.md

```diff
@@ -8,63 +8,73 @@
 
 ## Table of Contents
 
-- [Requirements](#requirements)
-- [Installation](#installation)
-  - [Platform Packages](#platform-packages)
+- [Installing](#installing)
+  - [Requirements](#requirements)
+  - [From sources](#from-sources)
   - [Godep](#godep)
+  - [Platform Packages](#platform-packages)
   - [Cross Compilation](#cross-compilation)
-- [Configuration](#configuration)
-- [Usage](#usage)
+  - [API keys](#api-keys)
+  - [Scripts](#scripts)
+- [Using](#using)
+  - [Hyphens](#hypens)
   - [Initializing](#initializing)
   - [De Initializing](#de-initializing)
-  - [End to End Encryption](#end-to-end-encryption)
   - [Traversal Depth](#traversal-depth)
+  - [Configuring General Settings](#configuring-general-settings)
+  - [Excluding And Including Objects](#excluding-and-including-objects)
+    - [Sample .driveignore with the exclude and include clauses combined](sample-.driveignore-with-the-exclude-and-include-clauses-combined)
   - [Pulling](#pulling)
+    - [Verifying Checksum] (#verifying-checksum)
     - [Exporting Docs](#exporting-docs)
   - [Pushing](#pushing)
+  - [Pulling And Pushing Notes](#pulling-and-pushing-notes)
+  - [Pulling And Pushing End to End Encryption](#pulling-and-pushing-end-to-end-encryption)
   - [Publishing](#publishing)
   - [Unpublishing](#unpublishing)
   - [Sharing and Emailing](#sharing-and-emailing)
   - [Unsharing](#unsharing)
+  - [Starring Or Unstarring](#starring-or-unstarring)
   - [Diffing](#diffing)
   - [Touching](#touching)
-  - [Trashing and Untrashing](#trashing-and-untrashing)
-  - [Emptying the Trash](#emptying-the-trash)
+  - [Trashing And Untrashing](#trashing-and-untrashing)
+  - [Emptying The Trash](#emptying-the-trash)
   - [Deleting](#deleting)
-  - [Listing Files](#listing-files)
-  - [Stating Files](#stating-files)
-  - [Retrieving md5 checksums](#retrieving-md5-checksums)
+  - [Listing](#listing)
+  - [Stating](#stating)
+  - [Printing URL](#printing-url)
   - [Editing Description](#editing-description)
-  - [FileId Retrieval](#fileid-retrieval)
-  - [New File](#new-file)
-  - [Quota](#quota)
-  - [Features](#features)
-  - [About](#about)
-  - [Help](#help)
-  - [Move](#move)
-  - [Rename](#rename)
-  - [Clashes](#clashes)
-  - [DriveIgnore](#driveignore)
-  - [DriveRC](#driverc)
-  - [DesktopEntry](#desktopentry)
+  - [Retrieving MD5 Checksums](#retrieving-md5-checksums)
+  - [Retrieving Id](#retrieving-id)
+  - [Retrieving Quota](#retrieving-quota)
+  - [Retrieving Features](#retrieving-features)
+  - [Creating](#creating)
+  - [Opening](#opening)
+  - [Copying](#copying)
+  - [Moving](#moving)
+  - [Renaming](#renaming)
   - [Command Aliases](#command-aliases)
-  - [Index Prune](#index-prune)
-  - [Url](#url)
-  - [Open](#open)
+  - [Detecting And Fixing Clashes](#detecting-and-fixing-clashes)
+  - [.desktop Files](#.desktop-files)
```

Comment:
Great addition of `.desktop files` but I think we forgot to include details about it.

Reply:
Please don't fight with me Son